Intersect.Editor/Forms/Editors/frmSpell.cs

```diff
@@ -213,6 +216,7 @@ private void InitLocalization()
             grpHotDot.Text = Strings.SpellEditor.hotdot;
             chkHOTDOT.Text = Strings.SpellEditor.ishotdot;
             lblTick.Text = Strings.SpellEditor.hotdottick;
+            lblOTanimationDisclaimer.Text = "(Set to " + Strings.General.none + " to keep same animation)";
```

Comment:
This entire thing needs to be translated

Reply:
I think I did this in a really dumb way - lemme know, but otherwise done